llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -821,12 +830,28 @@ InstrItinData<II_VMOV_D, [EmptyCycles<4>, InstrStage<1, [CM_WA_PORT]>],
 // However, Post-RA scheduling needs an exact itinerary, otherwise we might give
 // an optimistic latency to anti-dependencies.
 // See schedule/negative_latencies/bypass.mir for details.
-// FIXME: II_VMOV_W and II_VMOV_X always book both the vector and accumulator read and write ports.
+InstrItinData<II_VMOV_W, [SimpleCycle<CM_RM_PORT>, PrefixCycle<W_WM_PORT>, SimpleCycle<CM_WM_PORT>],
+              [2,1], [NoBypass, NoBypass]>,
+InstrItinData<II_VMOV_W_CM_CM, [SimpleCycle<CM_RM_PORT>, SimpleCycle<CM_WM_PORT>],
+              [2,1], [NoBypass, NoBypass]>,
+InstrItinData<II_VMOV_W_WMH_WMH, [EmptyCycles<1>, SimpleCycle<W_WM_PORT>],
+              [2,1], [NoBypass, NoBypass]>,
+InstrItinData<II_VMOV_W_WMH_CM, [SimpleCycle<CM_RM_PORT>, SimpleCycle<W_WM_PORT>],
+              [2,1], [NoBypass, NoBypass]>,
+InstrItinData<II_VMOV_W_CM_WMH, [EmptyCycles<1>, SimpleCycle<CM_WM_PORT>],
+              [2,1], [NoBypass, NoBypass]>,
+InstrItinData<II_VMOV_W_WML_CM, [SimpleCycle<CM_RM_PORT>, SimpleCycle<W_WM_PORT>],
+              [2,1], [VEC_Bypass, NoBypass]>,
+InstrItinData<II_VMOV_W_CM_WML, [EmptyCycles<1>, SimpleCycle<CM_WM_PORT>],
+              [2,1], [NoBypass, VEC_Bypass]>,
+InstrItinData<II_VMOV_W_WML_WMH, [EmptyCycles<1>, SimpleCycle<W_WM_PORT>],
+              [2,1], [VEC_Bypass, NoBypass]>,
+InstrItinData<II_VMOV_W_WMH_WML, [EmptyCycles<1>, SimpleCycle<W_WM_PORT>],
+              [2,1], [NoBypass, VEC_Bypass]>,
```

Comment:
I think MOV_W is a mov slot instruction, and can only use the MOV_Bypass for X and WL registers

Reply:
Thank you, have updated accordingly.